Extension/package.nls.json

```diff
@@ -239,7 +239,7 @@
     "c_cpp.configuration.legacyCompilerArgsBehavior.markdownDescription": "Enable pre-v1.10.0 behavior for how shell escaping is handled in compiler arg settings. Shell escaping is no longer expected or supported by default in arg arrays starting in v1.10.0.",
     "c_cpp.configuration.legacyCompilerArgsBehavior.deprecationMessage": "This setting is temporary to support transitioning to corrected behavior in v1.10.0.",
     "c_cpp.contributes.views.cppReferencesView.title": "C/C++: Other references results",
-    "c_cpp.contributes.views.SshTargetsView.title": "C/C++: SSH targets",
+    "c_cpp.contributes.views.SshTargetsView.title": { "message": "CppTools: SSH targets", "comment": [ "Do not localize `CppTools`." ]  },
```

Comment:
nit: We never capitalize the 'T' in 'tools'.  VS Code changes the casing anyway, but ...

Also, it seems the naming of the views is camelCase, so `sshTargetsView` would be better.